bot/connector-whatsapp-cloud/src/main/kotlin/database/repository/PayloadWhatsAppCloudMongoDAO.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2017/2021 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.connector.whatsapp.cloud.database.repository
+
+import ai.tock.bot.connector.whatsapp.cloud.database.model.PayloadWhatsAppCloud
+import ai.tock.shared.TOCK_BOT_DATABASE
+import ai.tock.shared.error
+import ai.tock.shared.injector
+import com.github.salomonbrys.kodein.instance
+import com.mongodb.client.MongoCollection
+import com.mongodb.client.MongoDatabase
+import com.mongodb.client.model.CreateCollectionOptions
+import com.mongodb.client.model.IndexOptions
+import com.mongodb.client.model.ReplaceOptions
+import mu.KotlinLogging
+import org.litote.kmongo.*
+import java.util.concurrent.TimeUnit
+
+object PayloadWhatsAppCloudMongoDAO: PayloadWhatsAppCloudDAO {
+
+    private const val COLLECTION_NAME = "sncv_whatsapp_payload"
```

Comment:
I would prefer to write 

private val collectionName = property("tock_whatsapp_payload", "whatsapp_payload")